### ISSUE TITLE
Log prob and alpha matrix

### ISSUE THREAD

**Issue by l4zyf9x** [Deepest-Project/AlignTTS#3]
See you guy have a great work on AlignTTS. I have some concerns about training. I have tried to reimplement MDN loss, Viterbi algorithm on Tensorflow, and training phase 0. The loss is similar to yours. I wanna check that log_prob and alpha actually look like? Could you guy give me some picture about log_prob and alpha. Here is some figure about loss, log_prob, and alpha

![image](https://user-images.githubusercontent.com/23411869/86443909-2386a180-bd3a-11ea-8957-0b59cd2cf54f.png)
![image](https://user-images.githubusercontent.com/23411869/86443966-39946200-bd3a-11ea-85c7-cc70a2829e10.png)
![image](https://user-images.githubusercontent.com/23411869/86444003-4b760500-bd3a-11ea-8ece-c0642566536e.png)
